src/bin/wasmer.rs

```diff
@@ -105,7 +105,22 @@ enum Backend {
 
 impl Backend {
     pub fn variants() -> &'static [&'static str] {
-        &["singlepass", "cranelift", "llvm"]
+        #[cfg(all(feature = "backend:singlepass", feature = "backend:llvm"))]
```

Comment:
What if this was 
```rust
&[
    #[cfg(feature = "backend:singlepass")]
    "singlepass",
    #[cfg(feature = "backend:cranelift")]
    "cranelift",
    ...
]
```

instead?

Reply:
I assumed that that wouldn't work due to the commas, but I can try it